src/events/events.controller.ts

```diff
@@ -71,22 +76,21 @@ export class EventsController {
       }),
     )
     { graffiti, points, type, occurred_at: occurredAt, url }: CreateEventDto,
-  ): Promise<SerializedEvent | null> {
+    @Res() res: Response,
+  ): Promise<void> {
     const user = await this.usersService.findByGraffitiOrThrow(graffiti);
 
-    const event = await this.eventsService.create({
-      type,
-      points,
-      occurredAt,
-      userId: user.id,
-      url,
-    });
-
-    if (!event) {
-      return null;
-    }
-
-    return serializedEventFromRecordWithMetadata(event);
+    await this.graphileWorkerService.addJob(
```

Comment:
Do we have any concern with the create event endpoint getting called very often? I know this was basically how the `deposit` jobs ran out of control. If this is a relatively infrequently used job, seems fine to me.

Reply:
This won't be used that often. It's used for manual events from Retool.